interface/ceed-basis.c

```diff
@@ -1059,6 +1174,32 @@ int CeedBasisGetGrad1D(CeedBasis basis, const CeedScalar **grad_1d) {
   return CEED_ERROR_SUCCESS;
 }
 
+/**
+  @brief Get divergence matrix of a CeedBasisHdiv
+  @param basis        CeedBasisHdiv
+  @param[out] div  Variable to store divergence matrix
+  @return An error code: 0 - success, otherwise - failure
+  @ref Advanced
+**/
+int CeedBasisGetDiv(CeedBasis basis, const CeedScalar **div) {
+
+  CeedFESpace FE_space = basis->basis_space;
+  switch (FE_space) {
```

Comment:
I think we can simplify/future proof this function a bit:
```suggestion
int CeedBasisGetDiv(CeedBasis basis, const CeedScalar **div) {
  if (!basis->div)
     return CeedError(basis->ceed, CEED_ERROR_MINOR,
                     "CeedBasis does not have divergence matrix.");
 
   *div = basis->div;
   return CEED_ERROR_SUCCESS;
 }
```